core/egov-localization/src/main/java/org/egov/domain/service/MessageService.java

```diff
@@ -104,13 +103,22 @@ public void bustCache() {
         return messages;
     }
     
-    public boolean createMessage(CreateMessagesRequest createMessagesRequest){
-    	messageRepository.deleteMessages(createMessagesRequest);
-   		boolean createStatus = messageRepository.createMessage(createMessagesRequest);
+    public boolean createMessage(String locale, String tenantId, List<Message> messageList){
+    	messageRepository.deleteMessages(locale, tenantId, messageList);
+   		boolean createStatus = messageRepository.createMessage(locale, tenantId, messageList);
    		if(createStatus){
    			return true;
    		}
    		return false; 
     }
-
+    
+    public boolean deleteMessage(String locale, String tenantId, List<Message> messageList){
```

Comment:
@darshannagesh Similar feedback here..Why is tenant and locale separate parameters?